whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/domain/NServerValidator.java

```diff
@@ -66,4 +72,19 @@ public ImmutableList<Action> getActions() {
     public ImmutableList<ObjectType> getTypes() {
         return TYPES;
     }
+
+
+    private void validateNserverCorrectPrefixes(RpslObject rpslObject, int prefixLength, boolean isIpv4){
+        if (hasRipeNserver(rpslObject) && hasIncorrectPrefixes(prefixLength, isIpv4)){
+            throw new BadRequestException();
+        }
```

Comment:
Can you add a message in updateContext and create a new error message in UpdateMessages. Just check the below lines. For example we do it for invalid hostname in the same class "UpdateMessages.hostNameMustEndWith(domain.getValue())"